tox.ini

```diff
@@ -1,7 +1,14 @@
 [tox]
-envlist = py37,build_docs
+envlist = py36,py37,py37-astropydev,py36-minimal,py38,py38-numpydev,py39,build_docs
```

Comment:
Thanks for adding in Python 3.9!  I'm also inclined to drop Python 3.6 support soon, though that's probably out of scope for this PR.